lib/ember_cli/runner.rb

```diff
@@ -32,5 +32,25 @@ def run!(command)
 
       true
     end
+
+    protected
+
+    attr_reader :env, :err_streams, :options, :out_streams
+
+    private
+
+    def write_to_out(output)
+      write(out_streams, output)
+    end
+
+    def write_to_err(output)
+      write(out_streams + err_streams, output)
```

Comment:
@thoughtbot/shell does it make sense to write errors to both `STDERR` and `STDOUT`?

I'm having trouble using `git blame` to find the commit that introduced (and justified) this behavior (this PR is only a refactoring of the behavior).

Is it against conventions to only log errors to `STDERR`?